recipes-bsp/u-boot/files/0009-u-boot-dts-remove-usb3.0-related-dts-configurations.patch

```diff
@@ -0,0 +1,78 @@
+From b736741e8b5db849b9c698a8bbdd2f3a3549c714 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Fri, 24 Sep 2021 11:50:29 +0800
+Subject: [PATCH 9/9] u-boot: dts : remove usb3.0 related dts configurations
+
+This is a known issue that usb3.0 can not work under u-boot.
+The usb3.0 port can only work at usb2.0 mode. So remove the usb3.0
+related dts configurations,and disable the usb3.0 serdes signal.
+
```

Comment:
This does not describe the effect of this change. I assume it's that the port will fall back to USB 2.0 mode and can thus be used this way.